base/workflow/R/start_model_runs.R

```diff
@@ -271,17 +279,10 @@ start_model_runs <- function(settings, write = TRUE, stop.on.error = TRUE) {
       }
       
       if (job_finished) {
-        
-        # Copy data back to local
-        if (!is_local) {
-          PEcAn.remote::remote.copy.from(
-            host = settings$host,
-            src = file.path(settings$host$outdir, run_id_string),
-            dst = settings$modeloutdir)
-        }
-        
+      
         # TODO check output log
         if (is_rabbitmq) {
+          #TODO: unclear to me if this path is local or remote.  If local, then needs to be moved outside of for loop after files are copied back to local.
           data <- readLines(file.path(jobids[run], "rabbitmq.out"))
           if (data[-1] == "ERROR") {
             msg <- paste("Run", run, "has an ERROR executing")
```

Comment:
@robkooper Do you have the ability to check if this still works?  I don't have the docker setup with rabbitmq on my machine and I can't tell if I've broken this part by pulling lines 274-282 outside of this for-loop.

Reply:
What if we did this with `remote.execute.cmd()` instead? I'm not actually sure which would be best for performance, but I suspect that would be more light-weight than copying the log file.

The thing I'm confused about is if the path to `rabbitmq.out` is really `"/<jobid>/rabbitmq.out"`.  Shouldn't it be in `settings$modeloutdir` somewhere?

Reply:
If you could check and tell me where rabbitmq.out ends up relative to `settings$outdir` or `settings$modeloutdir` that would be super helpful.